src/main/java/abyss/exception/AbyssException.java

```diff
@@ -1,6 +1,14 @@
 package abyss.exception;
 
+/**
+ * Represents an exception that occurs in Abyss.
+ */
 public class AbyssException extends Exception {
+    /**
+     * Constructor for AbyssException.
+     *
+     * @param message Error message.
+     */

```

Comment:
Perhaps javadoc comments for constructors are not needed since they are not methods?